core/src/main/scala/org/apache/gearpump/cluster/embedded/EmbeddedCluster.scala

```diff
@@ -36,27 +36,21 @@ import org.apache.gearpump.util.{LogUtil, Util}
  * Create a in-process cluster with single worker
  */
 class EmbeddedCluster(inputConfig: Config) {
-
-  private val workerCount: Int = 1
-  private var _master: ActorRef = null
-  private var _system: ActorSystem = null
-  private var _config: Config = null
-
   private val LOG = LogUtil.getLogger(getClass)
+  private val workerCount: Int = 1
+  var _master: ActorRef = _
```

Comment:
Do we have to expose these mutable variables ?  BTW, how about removing the prefix "_" since I don't see any conflicts here ?